src/test/regress/expected/failure_single_select.out

```diff
@@ -81,6 +78,7 @@ DETAIL:  server closed the connection unexpectedly
 (3 rows)
 
 COMMIT;
+ERROR:  failure on connection marked as essential: localhost:xxxxx
 -- some clean up
 UPDATE pg_dist_shard_placement SET shardstate = 1
```

Comment:
Pretty sure we can replace clean up with a check that there are no unhealthy shards